src/main/scala/is/hail/sparkextras/OrderedRDD.scala

```diff
@@ -490,9 +490,9 @@ class OrderedRDD[PK, K, V] private(rdd: RDD[(K, V)], val orderedPartitioner: Ord
     while (i < nPartitions) {
 
       val include = if (i == 0)
-        intervalArray.exists(_.start <= rangeBounds(0))
+        intervalArray.exists(_._1.start <= rangeBounds(0))
```

Comment:
IntervalTree is actually a map, so I changed it to implement Traversable[(Interval[T], U)] instead of Traversable[Interval[T]] (because I needed to iterate over it and  pull out the U in one spot).